internal/e2e/e2e_index_test.go

```diff
@@ -558,57 +559,23 @@ func TestDeleteOldShards(t *testing.T) {
 	}
 }
 
-func TestPartialSuccess(t *testing.T) {
-	dir := t.TempDir()
-
-	opts := Options{
-		IndexDir:    dir,
-		ShardMax:    1024,
-		SizeMax:     1 << 20,
-		Parallelism: 1,
-	}
-	opts.RepositoryDescription.Name = "repo"
-	opts.SetDefaults()
-
-	b, err := NewBuilder(opts)
-	if err != nil {
-		t.Fatalf("NewBuilder: %v", err)
-	}
-
-	for i := 0; i < 4; i++ {
-		nm := fmt.Sprintf("F%d", i)
-		_ = b.AddFile(nm, []byte(strings.Repeat("01234567\n", 128)))
-	}
-	b.buildError = fmt.Errorf("any error")
```

Comment:
This whole was moved to `index/builder_test.go` to keep the `buildError` variable private. It's also not really an end-to-end test.